cmd/powcfg/transform.go

```diff
@@ -0,0 +1,96 @@
+package main
```

Comment:
TL;DR:
- Iterate all storage configs
- Call the provided transformation
- If the transformation changed the storage config, save the new value
- No writes are done if we're `--dryrun`.